runhouse/servers/cluster_servlet.py

```diff
@@ -441,10 +444,24 @@ async def astatus(self):
             "system_disk_usage": disk_usage,
         }
         status_data = ResourceStatusData(**status_data)
-        return status_data
 
-    def status(self):
-        return sync_function(self.astatus)()
+        if send_to_den:
+            logger.debug("Sending cluster status to Den")
+            cluster_rns_address = config_cluster.get("name")
+            cluster_uri = rns_client.format_rns_address(cluster_rns_address)
+            api_server_url = config_cluster.get(
+                "api_server_url", rns_client.api_server_url
+            )
+            # returning both status data and the post response because they are both used in the periodic cluster check
+            # loop in the cluster servlet.
+            return status_data, await ClusterServlet.save_status_metrics_to_den(
```

Comment:
its unclear what `await ClusterServlet.save_status_metrics_to_den` returns,  can we first set this to a variable? 
also i think we can move all the logic from inside this if statement into that function

Reply:
actually maybe we can decouple these two actions? meaning status only handles returning the status data, and the send to den is separate